doctests/home-json.js

```diff
@@ -0,0 +1,149 @@
+// EXAMPLE: js_home_json
+// REMOVE_START
+import assert from 'assert';
+// REMOVE_END
+// STEP_START import
+import {
+    createClient,
+    SchemaFieldTypes,
+    AggregateGroupByReducers,
+    AggregateSteps,
+} from 'redis';
+// STEP_END
+
+// STEP_START connect
+const client = await createClient();
+await client.connect();
+// STEP_END
+// REMOVE_START
+try {
+    await client.ft.dropIndex('idx:users', { DD: true });
+} catch{}
```

Comment:
?!

Reply:
The reason for this is that dropIndex generally throws an error if you try to drop an index that doesn't exist (at least in the clients for other languages). It seemed to work OK without the try...catch locally, but I got an error from the CI when I made the PR. I assumed the try...catch was required and it fixed the error. If I'm doing this the wrong way or misinterpreting the build error then please let me know.